packages/x-components/src/x-modules/search/store/__tests__/actions.spec.ts

```diff
@@ -56,6 +57,20 @@ describe('testing search module actions', () => {
   });
 
   describe('fetchAndSaveSearchResponse', () => {
+    const spiedSearch = jest.spyOn(adapter, 'search');
```

Comment:
The `adapter.search` is already a jest spy because the adapter here is a mock. You can apply the expect directly to the adapter object